cfme/tests/containers/test_relationships_tables.py

```diff
@@ -49,19 +50,23 @@ def test_pods_rel(provider, rel):
         lambda ch: 'nginx' not in ch and not ch.startswith('metrics'),
         ui_pods)
 
-    for name in ui_pods_revised:
+    for name in random.shuffle(ui_pods_revised):
         obj = Pod(name, provider)
-
         val = obj.get_detail('Relationships', rel)
-        if val == '0':
-            continue
-        obj.click_element('Relationships', rel)
+        if val != '0':
+            break
```

Comment:
Me too. I think the best would actually be to create the stuff in openshift as a fixture of the test, but that's left for the future. Now I need to deal for huge running time (19 minutes for one test) of the tests quickly.

Perhaps https://pypi.python.org/pypi/pytest-randomly/1.0.0 can help with the tests repeatability by initialising the seed for every test.

I will report the savings after we fix some problems in the framework. We are facing this now:
```
            except MoveTargetOutOfBoundsException:  # This has become desperate now.
                raise exceptions.CannotScrollException(
>                   "Despite all the workarounds, scrolling to `{}` was unsuccessful.".format(loc))
E               CannotScrollException: Despite all the workarounds, scrolling to `//div[@id='list_grid']//table` was unsuccessful.

cfme/fixtures/pytest_selenium.py:693: CannotScrollException
```